src/ImmersedBoundaries/ImmersedBoundaries.jl

```diff
@@ -0,0 +1,82 @@
+module ImmersedBoundaries
+
+using Adapt
+
+using Oceananigans.Grids
+using Oceananigans.Operators
+using Oceananigans.Fields
+using Oceananigans.Utils
+using Oceananigans.TurbulenceClosures: AbstractTurbulenceClosure, time_discretization
+
+import Oceananigans.TurbulenceClosures:
+    viscous_flux_ux,
+    viscous_flux_uy,
+    viscous_flux_uz,
+    viscous_flux_vx,
+    viscous_flux_vy,
+    viscous_flux_vz,
+    viscous_flux_wx,
+    viscous_flux_wy,
+    viscous_flux_wz,
+    diffusive_flux_x,
+    diffusive_flux_y,
+    diffusive_flux_z,
+    κᶠᶜᶜ,
+    κᶜᶠᶜ,
+    κᶜᶜᶠ,
+    νᶜᶜᶜ,
+    νᶠᶠᶜ,
+    νᶜᶠᶠ,
+    νᶠᶜᶠ
```

Comment:
Should `Oceananigans.TurbulenceClosures` be exporting these since it's an internal module now?

Reply:
It could, but that wouldn't help here because we still need explicit `import` to extend these functions...